plans/dht/test/find_peers.go

```diff
@@ -169,5 +74,6 @@ Loop:
 
 	/// --- Ending the test
 
+	defer TearDown(ctx, runenv, watcher, writer)
```

Comment:
Given that we're defering this, we should run it immediately after setup.

Reply:
done